cfpb/consumerfinance.gov
# Ensure Elasticsearch index are lowercase

Lowercase our environment specific indexes to avoid problems with the rebuilding of them within jenkins.

## Changes

- automatically lowercase the index name to adhere to elasticsearch requirements.


## How to test this PR

1. Set deploy environment to something in caps like `STAGING`
2. Index should rebuild as expected.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
